### PR TITLE
Fix tools that have been merged due to wrong test results

### PR DESCRIPTION
**Pull request by bernt-matthias**
xref https://github.com/galaxyproject/planemo-ci-action/pull/47

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/abyss/abyss-pe.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <description>de novo sequence assembler</description>
     <macros>
         <token name="@TOOL_VERSION@">2.3.6</token>
-        <token name="@VERSION_SUFFIX@">0</token>
+        <token name="@VERSION_SUFFIX@">1</token>
```
Comment (bernt-matthias) [inline tools/abyss/abyss-pe.xml:5]:
```suggestion
        <token name="@VERSION_SUFFIX@">0</token>
```

File: tools/cnvkit/macros.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@VERSION_SUFFIX@">1</token>
```
Comment (bernt-matthias) [inline tools/cnvkit/macros.xml:2]:
```suggestion
    <token name="@VERSION_SUFFIX@">0</token>
```

File: tools/getorganelle/macros.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <macros>
     <token name="@TOOL_VERSION@">1.7.7.0</token>
-    <token name="@VERSION_SUFFIX@">1</token>
+    <token name="@VERSION_SUFFIX@">2</token>
```
Comment (bernt-matthias) [inline tools/getorganelle/macros.xml:3]:
```suggestion
    <token name="@VERSION_SUFFIX@">1</token>
```

File: tools/isoformswitchanalyzer/macros.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <macros>
     <token name="@TOOL_VERSION@">1.20.0</token>
-    <token name="@SUFFIX_VERSION@">1</token>
+    <token name="@SUFFIX_VERSION@">2</token>
```
Comment (bernt-matthias) [inline tools/isoformswitchanalyzer/macros.xml:3]:
```suggestion
    <token name="@SUFFIX_VERSION@">1</token>
```

**Review by bgruening — APPROVED** [review]

**Comment by mvdbeek** [comment]
Looks great to me, should we merge @bernt-matthias ?

**Comment by mvdbeek** [comment]
Ah, just saw you put it into ready to review :)

**Pull request merged**